x/ibc/02-client/keeper/client.go

```diff
@@ -71,8 +71,12 @@ func (k Keeper) UpdateClient(ctx sdk.Context, clientID string, header exported.H
 
 	switch clientType {
 	case exported.Tendermint:
+		trustedConsState, found := k.GetClientConsensusStateLTE(ctx, clientID, header.GetHeight())
+		if !found {
+			return nil, sdkerrors.Wrapf(types.ErrInvalidHeader, "could not find consensus state less than header height: %d to verify header against", header.GetHeight())
```

Comment:
I mean the reason it is erroring is because the header is invalid, there are no stored consensus states less than it. But i could change it if desired

Reply:
I can change it